Realm.xcodeproj/xcshareddata/xcschemes/TestHost.xcscheme

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes, this is intentional. See the commit message for 600b29f2b1cda45f25dccad5f39432b0d7882636.